README.md

```diff
@@ -27,7 +27,7 @@ Users of TAMS are insulated from the details of the underlying storage.
 
 ## Documentation
 - OpenAPI Specification: [TimeAddressableMediaStore.yaml](./api/TimeAddressableMediaStore.yaml)
-- Rendered Documentation: [TimeAddressableMediaStore.html](https://bbc.github.io/tams/TimeAddressableMediaStore.html)
+- Rendered Documentation: [https://bbc.github.io/tams](https://bbc.github.io/tams)
```

Comment:
Note that urls on lines 51 through to 61 will need to be updated.

Reply:
Good point, fixed